modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -208,13 +208,8 @@ const generatePopperOptions = (
     {
       name: 'offset',
       options: {
-        offset: ({reference}: {reference: Rect}) => {
-          // Skid menu along the edge of the button to account
-          // for fractional x-positioning of the button (e.g.,
-          // if the button is in a horizontally-centered modal)
-          // and to ensure proper alignment between the button
-          // and the menu
-          const skidding = reference.x % 1 >= 0.5 ? 1 : 0;
+        offset: () => {
+          const skidding = 0;
```

Comment:
We don't use `skidding` anymore. Skidding originally fixed a Chrome issue where some elements would render subpixel in some circumstances by not others. For instance, the centered dialog element in the Modal component would receive a `0.5px` subpixel offset if the window had an odd number for its width, but placing a Popper with the same subpixel offset would round to a whole number. Other browsers do not suffer this rounding issue.

The Modal now compensates by forcing an even number when centering the modal dialog element so we don't need this offset anymore. It didn't work in all circumstances anyways.